docs/embedding/sdk/questions.md

```diff
@@ -37,7 +37,7 @@ export default function App() {
 }
 ```
 
-You can pass parameter values to questions defined with SQL via `initialSqlParameters` prop, in the format of `{parameter_name: parameter_value}`. Learn more about [SQL parameters](../../questions/native-editor/sql-parameters.md).
+You can pass parameter values to questions defined with SQL via the `initialSqlParameters` prop, in the format of `{parameter_name: parameter_value}`. Learn more about [SQL parameters](../../questions/native-editor/sql-parameters.md). This works for both interactive and static questions.
```

Comment:
If we have a combined section for static and interactive questions, I don't think we need to specify this. I think it's already implied

Reply:
@oisincoveney This is in the "Embedding a static question" so it's a bit weird. Let me actually separate this into a separate section, cc @jeff-bruemmer

Reply:
updated in https://github.com/metabase/metabase/pull/50728/commits/ee5aa7953a163849e0812fd5b512627caa9c4dbd